src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_filtering.py

```diff
@@ -127,22 +127,23 @@ def __init__(self):
 
         # Stepping filters.
         pydevd_filters = os.getenv('PYDEVD_FILTERS', '')
-        if pydevd_filters.startswith('{'):
-            # dict(glob_pattern (str) -> exclude(True or False))
-            exclude_filters = []
-            for key, val in json.loads(pydevd_filters).items():
-                exclude_filters.append(ExcludeFilter(key, val, True))
-            self._exclude_filters = exclude_filters
-        else:
-            # A ';' separated list of strings with globs for the
-            # list of excludes.
-            filters = pydevd_filters.split(';')
-            pydev_log.debug("PYDEVD_FILTERS %s\n" % filters)
-            new_filters = []
-            for new_filter in filters:
-                if new_filter.strip():
-                    new_filters.append(ExcludeFilter(new_filter.strip(), True, True))
-            self._exclude_filters = new_filters
+        if pydevd_filters:
+            pydev_log.debug("PYDEVD_FILTERS %s\n" % (pydevd_filters,))
```

Comment:
why not doing debug level based logging here? I'm curious.

Reply:
Having `PYDEVD_FILTERS` as an environment variable is very uncommon, so, it doesn't seem useful to debug it in that case (nothing to see).